.github/workflows/ci.yml

```diff
@@ -15,8 +15,8 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python: ['3.7', '3.8', '3.9', '3.10']
-        django: ['3.2']
+        python: ['3.9', '3.10']
+        django: ['3.2', '4.1']
```

Comment:
please update `tox.ini` as well (I guess Django 4.0 can be removed too) and `setup.cfg` to update the trove classifiers

Reply:
Thanks good catch, done